fe/fe-core/src/main/java/org/apache/doris/planner/DistributedPlanner.java

```diff
@@ -60,9 +60,18 @@
     private final static Logger LOG = LogManager.getLogger(DistributedPlanner.class);
 
     private final PlannerContext ctx_;
+    private final long autoBroadcastJoinThreshold;
```

Comment:
Local variables within functions are better than member variables.

Reply:
put it to here because don't want to compute it more than once in one query

Reply:
Understood. Or put it into the global state ?

Reply:
> global state

do you mean GlobalState in Analyzer?

Reply:
Yes, I mean GlobalState in Analyzer.
We need to put it into query context. The only one param seems like context is Analyzer